sdk/program/src/message/legacy.rs

```diff
@@ -353,7 +353,7 @@ impl Message {
         self.program_position(i).is_some()
     }
 
-    pub fn is_writable(&self, i: usize) -> bool {
+    pub fn is_writable(&self, i: usize, demote_program_write_locks: bool) -> bool {
```

Comment:
Is this a backward compatibility issue?

Reply:
I don't think so, but tell me more about why you think there might be.

Reply:
I'm just thinking from an SDK ABI perspective

Reply:
Well, it used to have this ABI before I [removed it](https://github.com/solana-labs/solana/pull/18373). So if anything this makes it better :)

Reply:
I guess we are making exceptions to our backward compatibility policy for these cases?

Reply:
Urg, I guess so? Fortunately, I don't remember anyone noticing the last time we added or removed the bool.